setup.py

```diff
@@ -31,6 +31,7 @@
     include_package_data=True,
     install_requires=[
         "graphql-core-next>=1.0.4",
+        "python-multipart>=0.0.5",
```

Comment:
What is this?

Reply:
Starlette's dependency: https://github.com/encode/starlette/blob/master/starlette/requests.py#L182